src/Acme.UI/Views/Athletes/Athlete.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿
+@using ContentModels = Umbraco.Web.PublishedModels;
+@inherits Umbraco.Web.Mvc.UmbracoViewPage<Umbraco.Web.PublishedModels.Athlete>
+@using Umbraco.Web;
+
+@{
```

Comment:
I would suggest that you make a dedicated ViewModel if you need to derive more information from the original model.  Otherwise you are going to be doing this logic everywhere you use the data, Instantiate the ViewModel by passing in the Generated Model into its constructor.  Pass that view model to the View.